profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module

```diff
@@ -15,10 +15,7 @@ function nexteuropa_fields_formatters_field_formatter_info() {
       'description' => t('Displays as a button.'),
       'field types' => [
         'link_field',
-        'list_text',
         'taxonomy_term_reference',
```

Comment:
The formatter can be used with "Term reference" fields but why it cannot be used for "Entity reference" fields?

The module is available in the platform and is enabled by default.

Furthermore, it has also a link display like for term reference, via its "Label" formatter